src/index.js

```diff
@@ -225,6 +250,9 @@ function setAnalyticsIntegration(clientInstance) {
   return clientInstance;
 }
 
+// we check for document's existence to support server side rendering
+createFromScriptTag(create, document && document.querySelector('script[data-braintree-dropin-authorization]'));
```

Comment:
Since we have to look for this data attribute, I wanted to make sure it was unique. Is this overboard? Should it just be `[data-braintree-authorization]` or `[data-dropin-authorization]` or something else?

If we didn't support IE, we could just do `document.currentScript` :/